core/controllers/reader_test.py

```diff
@@ -205,6 +248,103 @@ def test_huge_answers(self):
             'a' * 1000500, 'Sorry, nope, we didn\'t get it', '')
 
 
+class ReaderClassifyTests(test_utils.GenericTestBase):
+    """Test reader.classify using the sample explorations.
+
+    Since the end to end tests cover correct classification,
+    ReaderClassifyTests is only checking which of the hard/soft/classifier
+    rules triggered on an input.
+    """
+
+    def setUp(self):
+        super(ReaderClassifyTests, self).setUp()
+        self.init_classify_inputs('16')
+
+    def init_classify_inputs(self, exploration_id):
+        exp_services.delete_demo(exploration_id)
+        exp_services.load_demo(exploration_id)
```

Comment:
So thinking through the pros and cons here, I'd like to keep the data somewhere separate from the test, but I see how this is not really made to be a demo exploration. Sean came up with moving the yaml into core/tests/data and loading it from there. I'm currently implementing that, but am tossing it out there to see what you think.